fe/lib/main.dart

```diff
@@ -21,11 +23,19 @@ class _MyAppState extends State<MyApp> with SingleTickerProviderStateMixin {
 
   late TabController _tabController;               // Enable tabs scroll horizontally
 
+  // Get data from server
+  getData() async{
+    var result = await http.get(Uri.parse("http://10.0.2.2:8000/api/vi/external/test"));
+    var result2 = json.decode(result.body);
+    print(result2);
```

Comment:
I think you should make this test before api made without printing something

Reply:
okay, thanks